chatkit-core/src/test/kotlin/com/pusher/chatkit/users/UserSubscriptionDispatcherTest.kt

```diff
@@ -38,12 +39,11 @@ object UserSubscriptionDispatcherTest : Spek({
 
     val simpleUser = UserApiType(
         id = "user1",
-        createdAt = "",
-        updatedAt = "",
+        createdAt = "2020-02-27T17:12:10Z",
+        updatedAt = "2020-02-27T17:12:10Z",
         name = "name",
-        avatarURL = null,
-        customData = null,
-        online = false
+        avatarUrl = null,
+        customData = null
     )
```

Comment:
we should have a test verifying CurrentUserReceived is dispatched here, i.e. after L78:
```kotlin
            it("then CurrentUserReceived is dispatched") {
                verify(exactly = 1) { dispatcher(CurrentUserReceived(
                    UserInternalType(
                        // expected test data
                    )
                )) }
            }
```

Reply:
👍 good idea! have added!